packages/wrangler/src/index.ts

```diff
@@ -1501,7 +1502,13 @@ export async function main(argv: string[]): Promise<void> {
 				logger.log(chalk.yellow(message));
 			}
 			const accountTag = (e as APIError)?.accountTag;
-			await whoami(accountTag);
+			let config: Config | undefined;
+			try {
+				config = await readConfig(wrangler.arguments, { hideWarnings: true });
+			} catch {
+				config = undefined;
+			}
+			await whoami(config, accountTag);
```

Comment:
Once again we need `config` in order to run `whoami` correctly. So we load it (very loosely) and then throw it away.